docs/contents/components/data-display/carousel/accessibility.ja.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Carousel
+description: "`Carousel`は、複数の要素をスライドショーのように表示するコンポーネントです。"
+package_name: "@yamada-ui/carousel"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`Carousel`は、アクセシビリティに関して[WAI-ARIA - Carousel (Slide Show or Image Rotator) Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/carousel/)に従います。
+
+## キーボード操作
```

Comment:
Instead of the word `"tab"`, all should be `"item"`.

The same applies to English.